backend/src/main/java/com/htecgroup/skynest/service/impl/UserServiceImpl.java

```diff
@@ -117,6 +119,14 @@ public void deleteUser(UUID uuid) {
 
   @Override
   public UserDto getUser(UUID uuid) {
```

Comment:
What is the purpose of this method?

Reply:
It is used for accessing and later changing user personal data. If he is worker he only can access his data, but admin can access every user data.

Reply:
Yes, that's right. If user has worker role but wants to see other user personal data he won't be able to do so.

Reply:
so this means according to the logic that if I am Admin , an Exception would be thrown? and that was not the wanted case, right?

Reply:
No, if admin access his or other user info exception would not be thrown. Only if worker try to access other user info exception is thrown.